### PR TITLE
#5 Implemented table with search and pagination, date picker, tabs 

### PR DESCRIPTION
**Pull request by SimerdeepSinghGrewal**
I've added tabs and table as per the requirement and figma design. For date picker, I've added two calendars to select date range but I was not sure about the third individual calendar.

### CODE REVIEW

**Comment by gorkem-bwl** [comment]
What is the third individual calendar? @SimerdeepSinghGrewal Can you show a screenshot?

**Comment by SimerdeepSinghGrewal** [comment]
Screenshot 2024-05-06 202132
the one on the right, like 2 calendars on the left and third one right single one

**Comment by uparkalau** [comment]
Your code looks good, but I would suggest making the folder structure clearer. Please reorganize your code following this example:
/src
  /components
    /Table
      Table.jsx
      TableStyles.css
    /DatePicker
      DatePicker.jsx
      DatePickerStyles.css
    /Tabs
      Tabs.jsx
      TabsStyles.jsx
  index.css // global styles
  App.jsx
  index.js

**Comment by SimerdeepSinghGrewal** [comment]
> When the button underneath that calendar is clicked



> Ah ok, I should have read that carefully. The one on the right is a single datepicker calendar. When the button underneath that calendar is clicked, corresponding calendar popups up, and the selected date is written on the button.

@gorkem-bwl Just to clarify, the two calendars are not connected, this is a single seperate datepicker calendar and when we click on the button that has the date written on it, the single calendar appears and the date text changes as per the selected date, does that sound correct?

**Comment by gorkem-bwl** [comment]
That is correct.

**Review by uparkalau — CHANGES_REQUESTED** [review]

**Review by uparkalau — APPROVED** [review]
Please merge it into develop branch.

**Review by uparkalau — APPROVED** [review]
You can merge it into the develop branch . We will create new task to refactor it.

**Review by uparkalau — APPROVED** [review]

**Review by uparkalau — DISMISSED** [review]

**Review by uparkalau — DISMISSED** [review]

File: frontend/src/components/Tabs/Tabs.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React, { useState } from 'react';
+import { Tab, Tabs } from '@mui/material';
+import './TabStyles.css';
+
+const MyTabs = () => {
```
[the remark this answers is not in the document]
Comment (SimerdeepSinghGrewal) [reply frontend/src/components/Tabs/Tabs.jsx:5]: renamed MyTabs to headingTabs but
using camelCase gives me this error
"[eslint]
src\components\Tabs\Tabs.jsx
  Line 6:29:  React Hook "useState" is called in function "headingTabs" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use"  react-hooks/rules-of-hooks

Search for the keywords to learn more about each error."

File: frontend/src/components/Table/TableStyles.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    display: flex;
+    align-items: center;
+}
+
+.paginationButtons .MuiButtonBase-root {
+    margin: 0 5px;
+}
+
+
+.direct_buttons{
```
[the remark this answers is not in the document]
Comment (SimerdeepSinghGrewal) [reply frontend/src/components/Table/TableStyles.css:77]: please clarify as I have used css as per the figma designs,

**Pull request closed without merging**